src/components/BarChart.js

```diff
@@ -168,7 +169,7 @@ const BarChart = React.createClass({
 
       transform = `translate(${x}, ${y})`;
     } else {
-      transform = 'translate(-100, -100)';
+      transform = 'translate(-1000, -1000)';
```

Comment:
This hides the tooltip further out of the SVG viewbox. When on a small screen, there were situations where it should show like a pixel of the tooltip.